literature.bib

```diff
@@ -7164,6 +7383,39 @@ @inproceedings{STKS:VaMoS12
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2011 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{PA:PVLDB11,
+	author = {Pawlik, Mateusz and Augsten, Nikolaus},
+	title = {{RTED: A Robust Algorithm for the Tree Edit Distance}},
+	journal = PVLDB,
+	volume = {5},
+	number = {4},
+	year = 2011,
+	issn = {2150--8097},
+	pages = {334--345},
+	numpages = {12},
+	url = {http://dx.doi.org/10.14778/2095686.2095692},
+	doi = {10.14778/2095686.2095692},
+	acmid = {2095692},
+	publisher = VLDB
```

Comment:
I guess no. The entries on Google Scholar, dblp and arxiv are so confusing and inconsistent.... I removed it now. I cannot even find a publisher because its only on arxiv.

Reply:
Look for other entries on arxiv in this database. I think I've handled such a case before.

Reply:
Did so. The other entries do not even have a publisher and some of them even are from the same journal. I will remove VLDB then.